tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1318,205 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
+var _ = Describe("{BasicBackupAndScheduleBackupUsingNamespaceLabel}", func() {
+	var (
+		err                        error
+		contexts                   []*scheduler.Context
+		appContexts                []*scheduler.Context
+		backupLocationUID          string
+		cloudCredUID               string
+		bkpNamespaces              []string
+		clusterUid                 string
+		backupName                 string
+		credName                   string
+		cloudCredUidList           []string
+		backupLocationName         string
+		nsLabels                   map[string]string
+		randomLabel                string
+		restoreName                string
+		periodicSchedulePolicyName string
+		periodicSchedulePolicyUid  string
+		restoreNames               []string
+		firstScheduleBackupName    string
+		scheduleNames              []string
+		scheduleBackupName         string
+		labeledSelector            map[string]string
+	)
+	labelSelectors := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("BasicBackupAndScheduleBackupUsingNamespaceLabel", "Schedule backup and restore, using namespace label and resource label", nil, 58041)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Schedule backup and restore, using namespace label and resource label", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(contexts)
+		})
+		Step("Generate and add labels to namespaces", func() {
+			log.InfoD("Generate and add labels to namespaces")
+			nsLabels = GenerateRandomLabels(3)
+			err = AddLabelsToMultipleNamespaces(nsLabels, bkpNamespaces)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Adding labels [%v] to namespaces [%v]", nsLabels, bkpNamespaces))
+		})
+		Step("Adding labels to resources", func() {
+			log.InfoD("Adding labels to resources")
+			labeledSelector = make(map[string]string)
+			labelKey := "backup-by-label"
+			labelValue := uuid.New()
+			for _, namespace := range bkpNamespaces {
+				pvcList, err := core.Instance().GetPersistentVolumeClaims(namespace, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching pvc list [%v] from namespace [%v]", pvcList, namespace))
+				for _, pvc := range pvcList.Items {
+					pvcPointer, err := core.Instance().GetPersistentVolumeClaim(pvc.Name, namespace)
```

Comment:
Change the variable name to something else. It should not say `pvcPointer `

Reply:
Done